src/Action.hs

```diff
@@ -144,14 +191,6 @@ instance GenArbitrary r => GenArbitrary (Action r) where
 
 -- * concrete monad type; user state
```

Comment:
actually, the 'user state' bit of the section header is outdated for some time already